packages/devtools/lib/src/ui/plotly.dart

```diff
@@ -0,0 +1,237 @@
+@JS()
```

Comment:
nit: here and elsewhere add the default copyright notice.

Reply:
Done.